pkg/controller/test/syncset/syncset.go

```diff
@@ -0,0 +1,94 @@
+package syncset
```

Comment:
IMO the test directory should be pkg/test. Keeping it under `controller` suggests that it should be used within `controllers`.

Reply:
I honestly don't care where it lands, but didn't we agree in the conversation to put it under `controllers`?

Reply:
We talked and apparently I misunderstood the original consensus of putting this under `pkg/test`. I'll move this.